docs/content/docs/plugins/oauth-provider.mdx

```diff
@@ -0,0 +1,1438 @@
+---
+title: oAuth 2.1 Provider
```

Comment:
Is the correct capitalisation not OAuth as supposed to oAuth?

Reply:
You are correct. Will use `OAuth` here